### DISCUSSION TITLE
Getting Black Screen with white dot upon start up

### DISCUSSION THREAD

**Discussion by swunglee** [i3/i3#5911] [Q&A]
I am trying to open up 6 evenly spaced terminal windows upon start-up. Whenever I start up the computer, it opens the 6 windows and is titled "Gnome Terminal," but the screen is black instead of opening up 6 terminal windows. It looks like the 3 screens in the "i3wm.org/docs/layout-saving.html" post. How do I get it to open the terminal application? Here is the workspace.json file for reference:
 
// vim:ts=4:sw=4:et;w
{
    // splith split container with 1 children
    "border": "normal",
    "floating": "auto_off",
    "layout": "splith",
    "marks": [],
    "percent": 1,
    "type": "con",
    "nodes": [
        {
            // splith split container with 3 children
            "border": "normal",
            "floating": "auto_off",
            "layout": "splith",
            "marks": [],
            "percent": 1,
            "type": "con",
            "nodes": [
                {
                    // splitv split container with 2 children
                    "border": "normal",
                    "floating": "auto_off",
                    "layout": "splitv",
                    "marks": [],
                    "percent": 0.333333333333333,
                    "type": "con",
                    "nodes": [
                        {
                            "border": "normal",
                            "current_border_width": 2,
                            "floating": "auto_off",
                            "geometry": {
                               "height": 531,
                               "width": 814,
                               "x": 0,
                               "y": 0
                            },
                            "marks": [],
                            "name": "cosmicmon@CosmicMon: ~",
                            "percent": 0.5,
                            "swallows": [
                               {
                                "class": "^Gnome\\-terminal$",
                                "instance": "^gnome\\-terminal\\-server$",
                                "machine": "^CosmicMon$",
                                "title": "^cosmicmon\\@CosmicMon\\:\\ \\~$",
                                "window_role": "^gnome\\-terminal\\-window\\-c70b5475\\-c6a4\\-4a38\\-9c89\\-4757b2cb1267$"
                               }
                            ],
                            "type": "con"
                        },
                        {
                            "border": "normal",
                            "current_border_width": 2,
                            "floating": "auto_off",
                            "geometry": {
                               "height": 531,
                               "width": 814,
                               "x": 0,
                               "y": 0
                            },
                            "marks": [],
                            "name": "cosmicmon@CosmicMon: ~",
                            "percent": 0.5,
                            "swallows": [
                               {
                                "class": "^Gnome\\-terminal$",
                                "instance": "^gnome\\-terminal\\-server$",
                                "machine": "^CosmicMon$",
                                "title": "^cosmicmon\\@CosmicMon\\:\\ \\~$",
                                "window_role": "^gnome\\-terminal\\-window\\-ac047776\\-5706\\-4a3d\\-920b\\-03ad3f394444$"
                               }
                            ],
                            "type": "con"
                        }
                    ]
                },
                {
                    // splitv split container with 2 children
                    "border": "normal",
                    "floating": "auto_off",
                    "layout": "splitv",
                    "marks": [],
                    "percent": 0.333333333333333,
                    "type": "con",
                    "nodes": [
                        {
                            "border": "normal",
                            "current_border_width": 2,
                            "floating": "auto_off",
                            "geometry": {
                               "height": 531,
                               "width": 814,
                                "x": 0,
                               "y": 0
                            },
                            "marks": [],
                            "name": "cosmicmon@CosmicMon: ~",
                            "percent": 0.5,
                            "swallows": [
                               {
                                "class": "^Gnome\\-terminal$",
                                "instance": "^gnome\\-terminal\\-server$",
                                "machine": "^CosmicMon$",
                                "title": "^cosmicmon\\@CosmicMon\\:\\ \\~$",
                                "window_role": "^gnome\\-terminal\\-window\\-ac047776\\-5706\\-4a3d\\-920b\\-03ad3f394444$"
                               }
                            ],
                            "type": "con"
                        }
                    ]
                },
                {
                    // splitv split container with 2 children
                    "border": "normal",
                    "floating": "auto_off",
                    "layout": "splitv",
                    "marks": [],
                    "percent": 0.333333333333333,
                    "type": "con",
                    "nodes": [
                        {
                            "border": "normal",
                            "current_border_width": 2,
                            "floating": "auto_off",
                            "geometry": {
                               "height": 531,
                               "width": 814,
                               "x": 0,
                               "y": 0
                            },
                            "marks": [],
                            "name": "cosmicmon@CosmicMon: ~",
                            "percent": 0.5,
                            "swallows": [
                               {
                                "class": "^Gnome\\-terminal$",
                                "instance": "^gnome\\-terminal\\-server$",
                                "machine": "^CosmicMon$",
                                "title": "^cosmicmon\\@CosmicMon\\:\\ \\~$",
                                "window_role": "^gnome\\-terminal\\-window\\-5ccff77d\\-3103\\-4dac\\-b460\\-459990c6580c$"
                               }
                            ],
                            "type": "con"
                        },
                        {
                            "border": "normal",
                            "current_border_width": 2,
                            "floating": "auto_off",
                            "geometry": {
                               "height": 531,
                               "width": 814,
                               "x": 0,
                               "y": 0
                            },
                            "marks": [],
                            "name": "cosmicmon@CosmicMon: ~",
                            "percent": 0.5,
                            "swallows": [
                               {
                                "class": "^Gnome\\-terminal$",
                                "instance": "^gnome\\-terminal\\-server$",
                                "machine": "^CosmicMon$",
                                "title": "^cosmicmon\\@CosmicMon\\:\\ \\~$",
                                "window_role": "^gnome\\-terminal\\-window\\-7a7823a2\\-1798\\-4dbb\\-a2a4\\-e4abb3bb1f88$"
                               }
                            ],
                            "type": "con"
                        }
                    ]
                },
                {
                    // splitv split container with 2 children
                    "border": "normal",
                    "floating": "auto_off",
                    "layout": "splitv",
                    "marks": [],
                    "percent": 0.333333333333333,
                    "type": "con",
                    "nodes": [
                        { 
                            "border": "normal",
                                "current_border_width": 2,
                            "floating": "auto_off",
                            "geometry": {
                               "height": 531,
                               "width": 814,
                               "x": 0,
                               "y": 0
                            },
                            "marks": [],
                            "name": "cosmicmon@CosmicMon: ~",
                            "percent": 0.5,
                            "swallows": [
                               {
                                "class": "^Gnome\\-terminal$",
                                "instance": "^gnome\\-terminal\\-server$",
                                "machine": "^CosmicMon$",
                                "title": "^cosmicmon\\@CosmicMon\\:\\ \\~$",
                                "window_role": "^gnome\\-terminal\\-window\\-5cc13088\\-59c3\\-4cb2\\-b237\\-0126c9a7d77b$"
                               }
                            ],
                            "type": "con"
                        },
                        {
                            "border": "normal",
                            "current_border_width": 2,
                            "floating": "auto_off",
                            "geometry": {
                               "height": 531,
                               "width": 814,
                               "x": 0,
                               "y": 0
                            },
                            "marks": [],
                            "name": "cosmicmon@CosmicMon: ~",
                            "percent": 0.5,
                            "swallows": [
                               {
                                "class": "^Gnome\\-terminal$",
                                "instance": "^gnome\\-terminal\\-server$",
                                "machine": "^CosmicMon$",
                                "title": "^cosmicmon\\@CosmicMon\\:\\ \\~$",
                                "window_role": "^gnome\\-terminal\\-window\\-fdddc16b\\-ee7f\\-4ee6\\-93e3\\-863c034ac03a$"
                               }
                            ],
                            "type": "con"
                        }
                    ]
                }
            ]
        }
    ]
}
~

**Comment by orestisfl** [comment]
i3's append_layout command doesn't actually run any program for you, just creates placeholders. You need to run gnome terminal 6 times yourself.

**Reply by swunglee** [reply]
Thank you for your reply. Is there any way to have it automatically run programs when the computer starts up? I'm ultimately trying to have the computer open to 6 chromium pages every time the computer is turned on, but was jus trying with terminal first.

**Reply by swunglee** [reply]
I've tried i3-resurrect. It only swallows my terminal window, not the Chromium windows. Is there another way to launch programs upon start-up?

**Reply by okraits** [reply]
Did you try this? https://i3wm.org/docs/userguide.html#_automatically_starting_applications_on_i3_startup

**Reply by orestisfl** [reply]
You can try this:
```
exec chromium --new-window
exec chromium --new-window
exec chromium --new-window
exec chromium --new-window
exec chromium --new-window
exec chromium --new-window
```
